nodejs/awsx/autoscaling/autoscaling.ts

```diff
@@ -219,12 +217,11 @@ function ifUndefined<T>(val: T | undefined, defVal: T) {
 function getCloudFormationTemplate(
     instanceName: string,
     instanceLaunchConfigurationId: pulumi.Output<string>,
-    subnetIds: pulumi.Input<string>[],
-    targetGroupArns: pulumi.Input<string>[],
+    subnetIds: pulumi.Input<pulumi.Input<string>[]>,
+    targetGroupArns: pulumi.Output<string>[],
```

Comment:
Why do we not accept this as an `Output` instead of an `Input`?  The latter should be more permissive.

Reply:
In general we shouldn't ever has `Output` on an input parameter.

Reply:
By the time we're at this helper function, the type is known to be an array of Outputs.  So this is just tightening things up.  (this is not a public api).